plugins/inputs/win_wmi/sample.conf

```diff
@@ -0,0 +1,8 @@
+[[inputs.win_wmi]]
+  name_prefix = "win_wmi_"
+  [[inputs.win_wmi.query]]
+    Namespace = "root\\cimv2"
+    ClassName = "Win32_Volume"
+    Properties = ["Name", "Capacity", "FreeSpace"]
+    Filter = 'NOT Name LIKE "\\\\?\\%"'
+    TagPropertiesInclude = ["Name"]
```

Comment:
Sample config does not contain the comments as seen in the readme file. There is also a missing line ending.

Furthermore, it does not represent the default configuration.

Reply:
I updated the README config example and the sample.conf to be identical.

>  it does not represent the default configuration

Can you elaborate on what this means?

Reply:
I think you did it the other way round. It is preferred to have some documentation/comments for each config property in the sample config.

You can ignore my other comment about default config as the properties aren't commented out (that would mean the suggested value represents the default value that would be used if the property isn't set).

Reply:
I believe this has all been addressed. Let me know if that is not the case.

Reply:
It is now. It wasn’t before.

Reply:
Great, thanks!